Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/InteractableToggleCollection.cs

```diff
@@ -23,7 +23,51 @@ public class InteractableToggleCollection : MonoBehaviour
         public Interactable[] ToggleList
         {
             get => toggleList;
-            set => toggleList = value;
+            set 
+            {
+                if (value == null)
+                {
+                    return;
+                }
+
+                if (toggleList != value)
+                {
+                    if (toggleList != null)
+                    {
+                        // Destroy all listeners on previous toggleList
+                        for (int i = 0; i < toggleActions.Count; ++i)
+                        {
+                            toggleList[i]?.OnClick.RemoveListener(toggleActions[i]);
+                        }
+                        listenersAdded = false;
+                        toggleActions.Clear();
+                    }
+
+                    // Set new list
+                    toggleList = value;
+
+                    // Add listeners to new list
+                    for (int i = 0; i < toggleList.Length; ++i)
+                    {
+                        int itemIndex = i;
+
+                        UnityAction setSelectionAction = () =>
+                        {
+                            SetSelection(itemIndex, true, false);
+                        };
+
+                        toggleActions.Add(setSelectionAction);
+
+                        toggleList[i].OnClick.AddListener(setSelectionAction);
+                        toggleList[i].CanDeselect = false;
+                    }
```

Comment:
This is redundant code with the Start(). Move to re-usable function

Reply:
Fixed, added AddSelectionListeners()